js/ad_manager_vast.js

```diff
@@ -282,9 +283,8 @@ OO.Ads.manager(function(_, $) {
      * @returns {boolean} Returns true if the root tag is valid otherwise it returns false.
      */
     this.isValidRootTagName = function(vastXML) {
-      var rootTagName = (vastXML && vastXML.firstChild) ? vastXML.firstChild.tagName || '' : '';
-      if (rootTagName.toUpperCase() != "VAST") {
-        OO.log("VAST: Invalid VAST XML for Tag Name: " + rootTagName);
+      if ($(vastXML).find("VAST").length === 0) {
+        OO.log("VAST: Invalid VAST XML");
```

Comment:
Got it. Can we use the new getVastRoot function to replace "$(vastXML).find("VAST")"?